src/Components/Scenes/Hydration/NewJob/NewJob.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Box } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import NewJobHeader from './NewJobHeader';
+
+const chromeStyles = makeStyles(theme => ({
+  container: {
+    position: 'fixed',
+    zIndex: 1101, // Positions container on top of original header bar
+    height: '100%',
+    width: '100%',
+    display: 'flex'
+  },
+  // Placeholder pattern until canvas is implemented
+  canvas: {
```

Comment:
I hope you rolled your own 100% on the CSS functions here

Reply:
Oh you know it